tox.ini

```diff
@@ -1,29 +1,22 @@
 [tox]
-envlist = coverage-clean, py27, py33, py34, py35, pypy, coverage-report
+envlist = flake8, py27, py34, py35, py36, py37, pypy, pypy3, coverage
 
 [testenv]
+usedevelop = true
 commands =
-    coverage run setup.py ftest
-setenv =
-  COVERAGE_FILE=.coverage.{envname}
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=80
 deps =
-    .[test]
     coverage
 
-[testenv:coverage-clean]
-deps = coverage
-setenv =
-  COVERAGE_FILE=.coverage
+[testenv:flake8]
+basepython = python3.6
```

Comment:
Any particular reason why coverage hardcodes basepython to 3.7 while flake8 hardcodes it to 3.6?

(Any particular reason to hardcode to any particular Python rather than using, say, `basepython = python3`?)

Reply:
My reason is a personal one: I do not have a `python3` on my computer, only the specific versions.